src/ts/createView.tsx

```diff
@@ -65,6 +68,17 @@ const createView = <TPluginState extends IPluginState<MatchType[]>>({
   overlayNode.classList.add("TyperighterPlugin__tooltip-overlay");
   logger.info("Typerighter plugin starting");
 
+  const pluginState = store.getState();
+
+  if (pluginState){
```

Comment:
I think that if we do want this (check the document eagerly on editor instantiation if this setting is enabled), we probably don't want to do it in `createView`, as this function won't fire as we expect in liveblogs.

Instead, perhaps worth adding to the `view` prop in the plugin – perhaps in another PR?

Reply:
I've moved to the `view` prop for now, but I will take it out of the PR if you feel strongly!

Reply:
No problem – I don't think it's working at present, as we need to call the bound commands, and at the moment we're importing an unbound one. I've pushed a fix in [b5b77f6](https://github.com/guardian/prosemirror-typerighter/pull/220/commits/b5b77f6a5ecf0444c0870c8c0f1a65c029a787a9) – I'll approve this, might be worth a quick look to verify that's correct before merging 👍